vue-app/src/views/RecipientRegistry.vue

```diff
@@ -1,85 +1,129 @@
 <template>
   <div class="recipients">
-    <h1 class="content-heading">Recipient registry</h1>
-    <div v-if="registryInfo" class="submit-project">
-      <div class="submit-project-info">
-        In order to become a recipient of funding, a project must go through a
-        review process.
-        <br />
-        It takes {{ formatDuration(registryInfo.challengePeriodDuration) }} and
-        requires a {{ formatAmount(registryInfo.deposit) }}
-        {{ registryInfo.depositToken }} security deposit.
+    <div v-if="isRecipientRegistryOwner">
+      <div class="title">
+        <div class="header">
+          <h2>Recipient registry</h2>
+        </div>
+        <div class="hr" />
+      </div>
+      <loader v-if="isLoading" />
+      <div v-else>
+        <table class="requests">
+          <thead>
+            <tr>
+              <th>Project</th>
+              <th>Request type</th>
+              <th>Status</th>
+              <th>Actions</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr
+              v-for="request in requests.slice().reverse()"
+              :key="request.transactionHash"
+            >
+              <td>
+                <div class="project-name">
+                  <a
+                    :href="request.metadata.thumbnailImageUrl"
+                    target="_blank"
+                    rel="noopener"
+                  >
+                    <img
+                      class="project-image"
+                      :src="request.metadata.thumbnailImageUrl"
+                    />
+                  </a>
+                  {{ request.metadata.name }}
+                  <router-link
+                    v-if="hasProjectLink(request)"
+                    :to="{
+                      name: 'project',
+                      params: { id: request.recipientId },
+                    }"
+                    >-></router-link
+                  >
+                </div>
+                <details class="project-details">
+                  <summary>More</summary>
+
+                  <div>
+                    <span
+                      >Transaction hash
+                      <img class="icon" width="16px" src="@/assets/copy.svg"
```

Comment:
Just curious if these different copy images in this column are supposed to have copy functionality

Reply:
Uhm...good question and good catch I think haha. Yes, we need that functionality working here. I forgot to review this part of the code.

- [ ] Add copy functionality